expertise/service/expertise.py

```diff
@@ -48,6 +48,17 @@ def __init__(self, client, config, logger):
         self.optional_fields = ['model', 'model_params', 'exclusion_inv', 'token', 'baseurl', 'baseurl_v2', 'paper_invitation', 'paper_id']
         self.path_fields = ['work_dir', 'scores_path', 'publications_path', 'submissions_path']
 
+        # For the optional fields, add camel case equivalents
+        camel_case_params = []
```

Comment:
instead of converting the field names, why don't you use the camel version directly?

Reply:
I think we can create a Config class that can be serialized from JSON object, similar to openreview-py classes.(Note, Invitation, etc)

Reply:
+1

Reply:
> I think we can create a Config class that can be serialized from JSON object, similar to openreview-py classes.(Note, Invitation, etc)

+1